src/main/java/io/confluent/idesidecar/restapi/cache/AdminClients.java

```diff
@@ -28,13 +29,14 @@ public AdminClients() {
    * @param clusterId    The cluster ID
    * @return The AdminClient
    */
-  public AdminClient getClient(String connectionId, String clusterId) {
+  public AdminClient getClient(String connectionId, String clusterId)
+      throws ClusterNotFoundException {
+    // Generate the Kafka admin client configuration
+    final var config = configurator.getAdminClientConfig(connectionId, clusterId, false);
```

Comment:
Moved this out of the lambda function since it now raises a checked exception `ClusterNotFoundException`.